tracker/view/show.py

```diff
@@ -569,3 +608,46 @@ def show_generated_advisory(advisory_id, raw=False):
     raw_asa = str(escape(raw_asa))
     raw_asa = advisory_extend_html(raw_asa, issues, package)
     return render_html_advisory(advisory=advisory, package=package, group=group, raw_asa=raw_asa, generated=True)
+
+
+@tracker.route('/advisory/<regex("{}"):advisory_id>/log'.format(advisory_regex[1:-1]), methods=['GET'])
+@tracker.route('/<regex("{}"):advisory_id>/log'.format(advisory_regex[1:-1]), methods=['GET'])
+def show_advisory_log(advisory_id, path=None):
+    advisory = (db.session.query(Advisory)
+                .filter(Advisory.id == advisory_id)
+                ).first()
+    if not advisory:
+        return not_found()
+
+    return render_template('log/advisory_log.html',
+                           title='{} - log'.format(advisory_id),
+                           advisory=advisory,
+                           can_watch_user_log=user_can_watch_user_log())
+
+
+# TODO: define permission to view this
```

Comment:
I'm in favor of first having it for "contributors" and later we can discuss about making it public